masq_lib/src/shared_schema.rs

```diff
@@ -367,6 +367,15 @@ pub fn shared_app(head: App<'static, 'static>) -> App<'static, 'static> {
             .max_values(1)
             .hidden(true),
     )
+    .arg(
+        Arg::with_name("fake-router-ip")
```

Comment:
I've once attempted (on your advice) to write tests that would verify some of the attributes set on the parameter in the Clap schema. Here it is marginal above all because it is a test parameter...

Reply:
Yeah...it's not clear to me at this point what value bringing the attributes of the `fake-router-ip` under test would have, since it's test code itself.